### DISCUSSION TITLE
Reaction confidence scores

### DISCUSSION THREAD

**Discussion by wshao1** [SysBioChalmers/Human-GEM#688] [Q&A] [answered]
Why are all the values set to 0 for the `rxnConfidenceScores` field of the MATLAB model? I am trying to use the mCADRE extraction algorithm with Human1, but it requires confidence scores.

**Comment by haowang-bioinfo** [comment, marked as answer]
> Why are all the values set to 0 for the rxnConfidenceScores field of the MATLAB model?

because this field hasn't been systematically curated so far

**Reply by mihai-sysbio** [reply]
Indeed, in the latest 1.16 release there are only 49 non-zero values for the confidence score field, and 13036 zeros.
